### PR TITLE
Deleted everything not related to SimpleSwap

### PR DESCRIPTION
**Pull request by Eknir**
I created a new branch [experimental](https://github.com/ethersphere/swap-swear-and-swindle/tree/experimental), which will be the new working branch for everything that is not related to SimpleSwap or not production ready. The intent is to, by time, add SoftSwap, Swap, Swear and Swindle back to Master, when we are confident about the quality of the contracts. 

To merge, first [Pull request #41](https://github.com/ethersphere/swap-swear-and-swindle/pull/41) should be reviewed and merged. 

See issue: https://github.com/ethersphere/swap-swear-and-swindle/issues/40

### CODE REVIEW

**Comment by ralph-pichler** [comment]
After the merge of #41 you should also add the removal of the _computePayout function from SimpleSwap to this PR.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 build
 node_modules
 .DS_Store
+package-lock.json
```
Comment (ralph-pichler) [inline .gitignore]: package-lock should remain committed to ensure every dev has the same exact dependencies for everytinh

File: package-lock.json [hunk 2]
```diff
@@ -1,624 +0,0 @@
-{
```
Comment (ralph-pichler) [inline package-lock.json]: this file should remain committed (see above)

Comment (Eknir) [reply package-lock.json]: done

File: .gitignore (hunk 1 above)
Comment (Eknir) [reply .gitignore]: Oke, thanks!

**Pull request merged**